ov_msckf/src/slam2.cpp

```diff
@@ -183,15 +188,16 @@ class slam2 : public plugin {
 		, sb{pb->lookup_impl<switchboard>()}
 		, _m_pose{sb->get_writer<pose_type>("slow_pose")}
 		, _m_imu_integrator_input{sb->get_writer<imu_integrator_input>("imu_integrator_input")}
-		, _m_begin{std::chrono::system_clock::now()}
+		, _m_rtc{pb->lookup_impl<RelativeClock>()}
+		// , _m_begin{_m_rtc->now()}
 		, open_vins_estimator{manager_params}
 		, imu_cam_buffer{nullptr}
 	{
-		_m_pose.put(_m_pose.allocate(
-			std::chrono::time_point<std::chrono::system_clock>{},
-			Eigen::Vector3f{0, 0, 0},
-			Eigen::Quaternionf{1, 0, 0, 0}
-		));
+		// _m_pose.put(_m_pose.allocate(
+		// 	_m_rtc->now(), 
+		// 	Eigen::Vector3f{0, 0, 0},
+		// 	Eigen::Quaternionf{1, 0, 0, 0}
+		// ));
```

Comment:
Why is this commented out?

What's surprising is that this branch still works. In that case, why was this required in the first place?

Reply:
I think it's because `_m_rtc->now()` might get called before the clock starts.

Reply:
Yeah, exactly, pose predict handles the lack of a pose. Also, we do something different [in Kimera](https://github.com/ILLIXR/Kimera-VIO/blob/master/examples/plugin.cpp#L52). We should just not publish anything in VIO's constructor.

Also, technically, the empty pose constructor on `data_format.hpp:157` should return an identity quaternion, not a zero quaternion.